repose-aggregator/src/docs/asciidoc/developer/performance.adoc

```diff
@@ -133,15 +133,15 @@ The directory structure of the performance tests is organized by `category` and
 That is, each file is located in a directory named after the specific `test` which is itself located in a directory named after the `category` of test.
 For example, the repose configuration files directory:
 
-* `repose-aggregator/tests/performance-tests/roles/repose/files/config/`
+* `repose-aggregator/tests/performance-tests/src/performanceTest/resources/roles/repose/files/config/`
```

Comment:
I super hate that we have to inject another three layers in this already crazy tree.
I don't immediately see a way around it, but I still don't like it.